.github/workflows/windows-builds.yml

```
@@ -0,0 +1,132 @@
+name: 'CI Windows On-Push'
+
+on:
+  push:
+    branches:
+      - github-actions
```

>As another note, this PR will need to be updated to define when it should run.

We run our Linux and Mac builds currently on pushes to and PR's against master (ignoring md-only changes), as well as on published GitHub releases:

```suggestion
on:
  push:
    branches:
      - master
    paths-ignore:
      - '**.md'
  pull_request:
    branches:
      - master
    paths-ignore:
      - '**.md'
  release:
    types:
      - published
```

---

Updated to reflect this, thanks!